src/langchain_google_spanner/vector_store.py

```diff
@@ -97,13 +102,43 @@ def __post_init__(self):
             raise ValueError("Index Columns can't be None")
 
 
+@dataclass
+class VectorSearchIndex:
```

Comment:
We can create this class as derived class of `SecondaryIndex` and inherit the `SecondaryIndex`